src/client.js

```diff
@@ -72,7 +73,8 @@ function Client(manager, name, config = {}) {
 
 	if (!Helper.config.public && client.config.log) {
 		if (Helper.config.messageStorage.includes("sqlite")) {
-			client.messageStorage.push(new MessageStorage(client));
+			client.messageProvider = new MessageStorage(client);
```

Comment:
The user can have multiple message providers active at once, so this won't work. I think we should sent `searchPossible` (or some equivalent) in the `init` event when a client connects. What do you think of that, @Nachtalb?

Reply:
At the moment the only message storages are sqlite and text from which sqlite is the only provider. I wanna implement Postgress later on end this would need a rework anyways. IMHO for the time being it's ok.